operator/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	ResponseDuration       uint64
 	PowDepth               uint64
 	Port                   int
+	Honest                 bool
```

Comment:
`RespondChallenges`

Reply:
Could you also give it a cmd line flag or is it more work to combine cmd line flags and config in the current implementation?

Reply:
The current implementation does not have flags, so this would be the only option with a flag.